src/Workleap.OpenApi.MSBuild/Exceptions/OpenApiTaskFailedException.cs

```diff
@@ -0,0 +1,8 @@
+namespace Workleap.OpenApi.MSBuild.Exceptions;
```

Comment:
I'm not sure we need to create a directory for this. I mean, everything else but this is at the root level

Reply:
(I don't actually really care however)